merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/GetSimAssociatedPlanDatasetsAction.java

```diff
@@ -0,0 +1,50 @@
+package gov.nasa.jpl.aerie.merlin.server.remotes.postgres;
+
+import gov.nasa.jpl.aerie.merlin.server.models.PlanId;
+import gov.nasa.jpl.aerie.merlin.server.models.SimulationDatasetId;
+import org.intellij.lang.annotations.Language;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static gov.nasa.jpl.aerie.merlin.server.remotes.postgres.PostgresParsers.parseOffset;
+
+/*package-local*/ final class GetSimAssociatedPlanDatasetsAction implements AutoCloseable {
```

Comment:
Minor and nonblocking, but I found this class name difficult to read, although I'm not sure how to improve it without making it even longer, ie `GetPlanDatasetsFromSimDatasetAction`